### PR TITLE
Add recipe for pandas_ml

### PR DESCRIPTION
**Pull request by souravsingh**
Fix for #2797

### CODE REVIEW

**Comment by souravsingh** [comment]
@sinhrks The recipe is getting a failure since it is not able to fetch ``LICENSE`` from the ``MANIFEST`` file.

**Comment by mariusvniekerk** [comment]
@souravsingh I've made a PR to the library to have this file included.  In the meantime you probably have to comment this out.

**Comment by sinhrks** [comment]
Sorry to delay. I've merged @mariusvniekerk 's PR. Should I release on PyPI?

**Comment by mariusvniekerk** [comment]
That would be great.

**Comment by souravsingh** [comment]
@sinhrks Has a new version for ``pandas_ml`` released?

**Comment by AbdealiJK** [comment]
Would love to have this conda package. Let me know if I can help on this.

**Comment by sinhrks** [comment]
Sorry for the delay. I've uploaded the new version which should include the license.

**Comment by souravsingh** [comment]
I have made the necessary fixes for the recipe. It should be ready to merge once the Travis tests pass.

File: recipes/pandas_ml/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas >=0.17.0
+    - enum34
```
Comment (AbdealiJK) [inline recipes/pandas_ml/meta.yaml]: enum34 does not seem to be a `build` requirement ... just a run-time requirement

File: recipes/pandas_ml/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  run:
+    - python
+    - pandas >=0.17.0
+    - enum34
```
Comment (AbdealiJK) [inline recipes/pandas_ml/meta.yaml]: It enum34 needed for python3.4 onward ? I think this should only be installed for earlier versions of python

File: recipes/pandas_ml/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    - pandas_ml.snsaccessors
+    - pandas_ml.snsaccessors.test
+    - pandas_ml.test
+    - pandas_ml.util
+    - pandas_ml.xgboost
+    - pandas_ml.xgboost.test
+    
+about:
+  home: https://github.com/pandas-ml/pandas-ml
+  license: BSD
```
Comment (AbdealiJK) [inline recipes/pandas_ml/meta.yaml]: `BSD 3-clause`

File: recipes/pandas_ml/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pandas >=0.17.0
+  run:
+    - python
+    - pandas >=0.17.0
+    - enum34
```
Comment (mwcraig) [inline recipes/pandas_ml/meta.yaml]: `enum34` should only be a dependency for python <3.4, i.e. for python 2 on conda-forge. Adding a selector to this requirement should fix the build failures.

File: recipes/pandas_ml/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    - pandas_ml.snsaccessors
+    - pandas_ml.snsaccessors.test
+    - pandas_ml.test
+    - pandas_ml.util
+    - pandas_ml.xgboost
+    - pandas_ml.xgboost.test
+    
+about:
+  home: https://github.com/pandas-ml/pandas-ml
+  license: BSD 3-clause
```
Comment (mwcraig) [inline recipes/pandas_ml/meta.yaml]: Please also add:

```
license_family: BSD
```

Comment (souravsingh) [reply recipes/pandas_ml/meta.yaml]: Done

**Comment by souravsingh** [comment]
All tests pass for the current commit. I think the PR is ready to merge, unless someone wants to add a comment.

**Comment by mwcraig** [comment]
Thanks, @souravsingh!

**Pull request merged**